framework/wazuh/core/cluster/worker.py

```diff
@@ -597,43 +597,44 @@ async def general_agent_sync_task(self, sync_object, timer, sleep_interval):
 
             await asyncio.sleep(sleep_interval)
 
-    async def sync_extra_valid(self, extra_valid: Dict):
+    async def sync_extra_valid(self, _: Dict):
```

Comment:
Here we can leave the extra_valid word, not using the function is enough. Also by eliminating this we make the function not so clear.